Kiosk/Storyboards/Fulfillment.storyboard

```diff
@@ -1680,15 +1680,23 @@
                         <rect key="frame" x="0.0" y="0.0" width="449" height="485"/>
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
+                            <button opaque="NO" contentMode="left" fixedFrame="YES" contentHorizontalAlignment="leading" contentVerticalAlignment="center" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="R87-pJ-OQN">
+                                <rect key="frame" x="0.0" y="86" width="44" height="44"/>
+                                <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
+                                <state key="normal" image="checkmark_unchecked"/>
+                                <connections>
+                                    <action selector="conditionsOfSaleCheckmarkTapped:" destination="q7z-ij-DNu" eventType="touchUpInside" id="gPa-dm-9Dt"/>
+                                </connections>
+                            </button>
```

Comment:
Was this done via the storyboard editor?

Reply:
That's right – I didn't even look at the text the diff until I opened the PR.